src/main/java/org/mskcc/cmo/publisher/pipeline/limsrest/LimsRequestWriter.java

```diff
@@ -0,0 +1,59 @@
+package org.mskcc.cmo.publisher.pipeline.limsrest;
+
+import com.google.gson.Gson;
+import java.util.List;
+import java.util.Map;
+import org.apache.log4j.Logger;
+import org.mskcc.cmo.messaging.Gateway;
+import org.springframework.batch.item.ExecutionContext;
+import org.springframework.batch.item.ItemStreamException;
+import org.springframework.batch.item.ItemStreamWriter;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+
+/**
+ *
+ * @author ochoaa
+ */
+public class LimsRequestWriter implements ItemStreamWriter<Map<String, Object>> {
+
+    @Autowired
+    private Gateway messagingGateway;
+
+    @Value("${igo.new_request_topic}")
```

Comment:
In the future we may want to use this tool to publish to other topics, you may just want to call this destination_topic or something.

Reply:
renamed to `lims.publisher_topic`